src/arkit.cs

```diff
@@ -62,6 +62,7 @@ public enum ARErrorCode : nint {
 		SensorFailed = 102,
 		CameraUnauthorized = 103,
 		WorldTrackingFailed = 200,
+		InvalidReferenceImage = 300
```

Comment:
minor (don't update PR just for that): add a `,` for new enum members
this makes the diff smaller for the next addition :)

Reply:
You're right, I forgot it this time